### PR TITLE
german translation of code editors

### PR DESCRIPTION
**Pull request by higoka**

### CODE REVIEW

File: 1-js/01-getting-started/3-code-editors/article.md [hunk 1]
```diff
@@ -1,46 +1,46 @@
...
+## Lass uns nicht streiten
 
-The editors in the lists above are those that either I or my friends whom I consider good developers have been using for a long time and are happy with.
+Die Editoren in den obigen Listen sind diejenigen, die ich oder meine Freunde, die ich für gute Entwickler halte, schon seit langer Zeit verwenden und mit denen ich zufrieden bin.
 
-There are other great editors in our big world. Please choose the one you like the most.
+Es gibt andere großartige Editoren in unserer großen Welt. Bitte wählen Sie den aus, der Ihnen am besten gefällt.
 
-The choice of an editor, like any other tool, is individual and depends on your projects, habits, and personal preferences.
+Die Wahl eines Editors ist wie jedes andere Tool individuell und hängt von Ihren Projekten, Gewohnheiten und persönlichen Vorlieben ab.
```
Comment (christianhegedues) [inline 1-js/01-getting-started/3-code-editors/article.md]:
```suggestion
Die Wahl eines Editors ist, wie jedes andere Tool, individuell und hängt von Ihren Projekten, Gewohnheiten und persönlichen Vorlieben ab.
```

File: 1-js/01-getting-started/3-code-editors/article.md [hunk 2]
```diff
@@ -1,46 +1,46 @@
...
-An IDE loads the project (which can be many files), allows navigation between files, provides autocompletion based on the whole project (not just the open file), and integrates with a version management system (like [git](https://git-scm.com/)), a testing environment, and other "project-level" stuff.
+Eine IDE lädt das Projekt (das aus vielen Dateien bestehen kann), ermöglicht das Navigieren zwischen Dateien, stellt die automatische Vervollständigung basierend auf dem gesamten Projekt (nicht nur der geöffneten Datei) bereit und lässt sich in ein Versionsverwaltungssystem (wie [git](https://git-scm.com/)), eine Testumgebung und andere Dinge auf "Projektebene".
 
-If you haven't selected an IDE yet, consider the following options:
+Wenn Sie noch keine IDE ausgewählt haben, prüfen Sie die folgenden Optionen:
 
-- [Visual Studio Code](https://code.visualstudio.com/) (cross-platform, free).
-- [WebStorm](http://www.jetbrains.com/webstorm/) (cross-platform, paid).
+- [Visual Studio Code](https://code.visualstudio.com/) (plattformübergreifend, kostenlos).
+- [WebStorm](http://www.jetbrains.com/webstorm/) (plattformübergreifend, bezahlt).
```
Comment (christianhegedues) [inline 1-js/01-getting-started/3-code-editors/article.md]:
```suggestion
- [WebStorm](http://www.jetbrains.com/webstorm/) (plattformübergreifend, kostenpflichtig).
```

File: 1-js/01-getting-started/3-code-editors/article.md [hunk 3]
```diff
@@ -1,46 +1,46 @@
...
+Der Begriff [IDE](https://de.wikipedia.org/wiki/Integrierte_Entwicklungsumgebung) (Integrated Development Environment) bezieht sich auf einen leistungsstarken Editor mit vielen Funktionen, der normalerweise ein "ganzes Projekt" bearbeitet. Wie der Name schon sagt, handelt es sich nicht nur um einen Editor, sondern um eine umfassende "Entwicklungsumgebung".
 
-An IDE loads the project (which can be many files), allows navigation between files, provides autocompletion based on the whole project (not just the open file), and integrates with a version management system (like [git](https://git-scm.com/)), a testing environment, and other "project-level" stuff.
+Eine IDE lädt das Projekt (das aus vielen Dateien bestehen kann), ermöglicht das Navigieren zwischen Dateien, stellt die automatische Vervollständigung basierend auf dem gesamten Projekt (nicht nur der geöffneten Datei) bereit und lässt sich in ein Versionsverwaltungssystem (wie [git](https://git-scm.com/)), eine Testumgebung und andere Dinge auf "Projektebene".
```
Comment (christianhegedues) [inline 1-js/01-getting-started/3-code-editors/article.md]:
```suggestion
Eine IDE lädt das Projekt (das aus vielen Dateien bestehen kann), ermöglicht das Navigieren zwischen Dateien, stellt die automatische Vervollständigung basierend auf dem gesamten Projekt (nicht nur der geöffneten Datei) bereit und lässt sich in ein Versionsverwaltungssystem (wie [git](https://git-scm.com/)), eine Testumgebung und andere Dinge auf "Projektebene" integrieren.
```

File: 1-js/01-getting-started/3-code-editors/article.md [hunk 4]
```diff
@@ -1,46 +1,46 @@
...
-If you haven't selected an IDE yet, consider the following options:
+Wenn Sie noch keine IDE ausgewählt haben, prüfen Sie die folgenden Optionen:
 
-- [Visual Studio Code](https://code.visualstudio.com/) (cross-platform, free).
-- [WebStorm](http://www.jetbrains.com/webstorm/) (cross-platform, paid).
+- [Visual Studio Code](https://code.visualstudio.com/) (plattformübergreifend, kostenlos).
+- [WebStorm](http://www.jetbrains.com/webstorm/) (plattformübergreifend, bezahlt).
 
-For Windows, there's also "Visual Studio", not to be confused with "Visual Studio Code". "Visual Studio" is a paid and mighty Windows-only editor, well-suited for the .NET platform. It's also good at JavaScript. There's also a free version [Visual Studio Community](https://www.visualstudio.com/vs/community/).
+Für Windows gibt es auch "Visual Studio", nicht zu verwechseln mit "Visual Studio Code". "Visual Studio" ist ein kostenpflichtiger und mächtiger Windows-Editor, der für die .NET-Plattform gut geeignet ist. Es ist auch gut in JavaScript. Es gibt auch eine kostenlose Version: [Visual Studio Community](https://www.visualstudio.com/vs/community/).
```
Comment (christianhegedues) [inline 1-js/01-getting-started/3-code-editors/article.md]:
```suggestion
Für Windows gibt es auch "Visual Studio", nicht zu verwechseln mit "Visual Studio Code". "Visual Studio" ist ein kostenpflichtiger und mächtiger Windows-Editor, der für die .NET-Plattform zugeschnitten ist. Er ist auch für JavaScript geeignet. Es gibt auch eine kostenlose Version: [Visual Studio Community](https://www.visualstudio.com/vs/community/).
```

File: 1-js/01-getting-started/3-code-editors/article.md [hunk 5]
```diff
@@ -1,46 +1,46 @@
...
 
-Many IDEs are paid, but have a trial period. Their cost is usually negligible compared to a qualified developer's salary, so just choose the best one for you.
+Viele IDEs werden bezahlt, haben aber eine Probezeit. Ihre Kosten sind im Vergleich zum Gehalt eines qualifizierten Entwicklers in der Regel vernachlässigbar. Wählen Sie also einfach das beste für Sie aus.
```
Comment (christianhegedues) [inline 1-js/01-getting-started/3-code-editors/article.md]:
```suggestion
Viele IDEs sind kostenpflichtig, haben aber eine Testphase. Ihre Kosten sind im Vergleich zum Gehalt eines qualifizierten Entwicklers normalerweise vernachlässigbar, wählen Sie also einfach den Besten für sich aus.
```

File: 1-js/01-getting-started/3-code-editors/article.md [hunk 6]
```diff
@@ -1,46 +1,46 @@
...
-- [Atom](https://atom.io/) (cross-platform, free).
-- [Visual Studio Code](https://code.visualstudio.com/) (cross-platform, free).
-- [Sublime Text](http://www.sublimetext.com) (cross-platform, shareware).
-- [Notepad++](https://notepad-plus-plus.org/) (Windows, free).
-- [Vim](http://www.vim.org/) and [Emacs](https://www.gnu.org/software/emacs/) are also cool if you know how to use them.
+- [Atom](https://atom.io/) (plattformübergreifend, kostenlos).
+- [Visual Studio Code](https://code.visualstudio.com/) (plattformübergreifend, kostenlos).
+- [Sublime Text](http://www.sublimetext.com) (plattformübergreifend, Shareware).
+- [Notepad++](https://notepad-plus-plus.org/) (Windows, kostenlos).
+- [Vim](http://www.vim.org/) und [Emacs](https://www.gnu.org/software/emacs/) sind auch cool, wenn du weißt, wie man sie benutzt.
```
Comment (christianhegedues) [inline 1-js/01-getting-started/3-code-editors/article.md]:
```suggestion
- [Vim](http://www.vim.org/) und [Emacs](https://www.gnu.org/software/emacs/) sind auch cool, wenn Sie wissen, wie man sie benutzt.
```

File: 1-js/01-getting-started/3-code-editors/article.md [hunk 7]
```diff
@@ -1,46 +1,46 @@
...
 
-They are mainly used to open and edit a file instantly.
+Sie werden hauptsächlich zum sofortigen Öffnen und Bearbeiten einer Datei verwendet.
 
-The main difference between a "lightweight editor" and an "IDE" is that an IDE works on a project-level, so it loads much more data on start, analyzes the project structure if needed and so on. A lightweight editor is much faster if we need only one file.
+Der Hauptunterschied zwischen einem "Lightweight-Editor" und einer "IDE" besteht darin, dass eine IDE auf Projektebene arbeitet, also beim Start viel mehr Daten lädt, die Projektstruktur bei Bedarf analysiert und so weiter. Ein Lightweight-Editor ist viel schneller, wenn wir nur eine Datei benötigen.
 
-In practice, lightweight editors may have a lot of plugins including directory-level syntax analyzers and autocompleters, so there's no strict border between a lightweight editor and an IDE.
+In der Praxis verfügen Lightweight-Editoren möglicherweise über eine Vielzahl von Plug-ins, darunter Syntaxanalysatoren und Autocompleter auf Verzeichnisebene. Daher gibt es keine strikte Grenze zwischen einem Lightweight-Editor und einer IDE.
```
Comment (christianhegedues) [inline 1-js/01-getting-started/3-code-editors/article.md]:
```suggestion
In der Praxis verfügen Lightweight-Editoren möglicherweise über eine Vielzahl von Plug-ins, darunter Syntaxanalyse und Autovervollständigung auf Verzeichnisebene. Daher gibt es keine exakte Grenze zwischen einem Lightweight-Editor und einer IDE.
```

File: 1-js/01-getting-started/3-code-editors/article.md [hunk 8]
```diff
@@ -1,46 +1,46 @@
...
+Der Begriff [IDE](https://de.wikipedia.org/wiki/Integrierte_Entwicklungsumgebung) (Integrated Development Environment) bezieht sich auf einen leistungsstarken Editor mit vielen Funktionen, der normalerweise ein "ganzes Projekt" bearbeitet. Wie der Name schon sagt, handelt es sich nicht nur um einen Editor, sondern um eine umfassende "Entwicklungsumgebung".
 
-An IDE loads the project (which can be many files), allows navigation between files, provides autocompletion based on the whole project (not just the open file), and integrates with a version management system (like [git](https://git-scm.com/)), a testing environment, and other "project-level" stuff.
+Eine IDE lädt das Projekt (das aus vielen Dateien bestehen kann), ermöglicht das Navigieren zwischen Dateien, stellt die automatische Vervollständigung basierend auf dem gesamten Projekt (nicht nur der geöffneten Datei) bereit und lässt sich in ein Versionsverwaltungssystem (wie [git](https://git-scm.com/)), eine Testumgebung und andere Dinge auf "Projektebene" integrieren.
 
-If you haven't selected an IDE yet, consider the following options:
+Wenn Sie noch keine IDE ausgewählt haben, prüfen Sie die folgenden Optionen:
```
Comment (1H0) [inline 1-js/01-getting-started/3-code-editors/article.md]:
```suggestion
Wenn du noch keine IDE ausgewählt haben, prüfen Sie die folgenden Optionen:
```

File: 1-js/01-getting-started/3-code-editors/article.md [hunk 9]
```diff
@@ -1,46 +1,46 @@
...
-- [Atom](https://atom.io/) (cross-platform, free).
-- [Visual Studio Code](https://code.visualstudio.com/) (cross-platform, free).
-- [Sublime Text](http://www.sublimetext.com) (cross-platform, shareware).
-- [Notepad++](https://notepad-plus-plus.org/) (Windows, free).
-- [Vim](http://www.vim.org/) and [Emacs](https://www.gnu.org/software/emacs/) are also cool if you know how to use them.
+- [Atom](https://atom.io/) (plattformübergreifend, kostenlos).
+- [Visual Studio Code](https://code.visualstudio.com/) (plattformübergreifend, kostenlos).
+- [Sublime Text](http://www.sublimetext.com) (plattformübergreifend, Shareware).
+- [Notepad++](https://notepad-plus-plus.org/) (Windows, kostenlos).
+- [Vim](http://www.vim.org/) und [Emacs](https://www.gnu.org/software/emacs/) sind auch cool, wenn Sie wissen, wie man sie benutzt.
```
Comment (1H0) [inline 1-js/01-getting-started/3-code-editors/article.md]:
```suggestion
- [Vim](http://www.vim.org/) und [Emacs](https://www.gnu.org/software/emacs/) sind auch cool, wenn du weisst, wie man sie benutzt.
```

File: 1-js/01-getting-started/3-code-editors/article.md [hunk 10]
```diff
@@ -1,46 +1,46 @@
...
 
-## Let's not argue
+## Lass uns nicht streiten
 
-The editors in the lists above are those that either I or my friends whom I consider good developers have been using for a long time and are happy with.
+Die Editoren in den obigen Listen sind diejenigen, die ich oder meine Freunde, die ich für gute Entwickler halte, schon seit langer Zeit verwenden und mit denen ich zufrieden bin.
```
Comment (1H0) [inline 1-js/01-getting-started/3-code-editors/article.md]:
```suggestion
Die Editoren in den obigen Listen sind diejenigen, die ich oder meine Freunde, die ich für gute Entwickler halte, schon seit langer Zeit verwenden und mit denen wir zufrieden sind.
```

File: 1-js/01-getting-started/3-code-editors/article.md [hunk 11]
```diff
@@ -1,46 +1,46 @@
...
 
-There are other great editors in our big world. Please choose the one you like the most.
+Es gibt andere großartige Editoren in unserer großen Welt. Bitte wählen Sie den aus, der Ihnen am besten gefällt.
```
Comment (1H0) [inline 1-js/01-getting-started/3-code-editors/article.md]:
```suggestion
Es gibt andere großartige Editoren in unserer großen Welt. Bitte wähle den aus, der dir am besten gefällt.
```

File: 1-js/01-getting-started/3-code-editors/article.md [hunk 12]
```diff
@@ -1,46 +1,46 @@
...
 
-The choice of an editor, like any other tool, is individual and depends on your projects, habits, and personal preferences.
+Die Wahl eines Editors ist, wie jedes andere Tool, individuell und hängt von Ihren Projekten, Gewohnheiten und persönlichen Vorlieben ab.
```
Comment (1H0) [inline 1-js/01-getting-started/3-code-editors/article.md]:
```suggestion
Die Wahl eines Editors ist, wie jedes andere Tool, individuell und hängt von deinen Projekten, Gewohnheiten und persönlichen Vorlieben ab.
```

**Comment by 1H0** [comment]
Just saw that the topic was not reserved by you and that @MichFriee reserved and translated the page in another PR. Please make sure to always check first, if the article isn't already taken and to reserve it first...

Ok when I close this PR and you take another article?

File: 1-js/01-getting-started/3-code-editors/article.md [hunk 13]
```diff
@@ -1,46 +1,46 @@
...
+Der Begriff [IDE](https://de.wikipedia.org/wiki/Integrierte_Entwicklungsumgebung) (Integrated Development Environment) bezieht sich auf einen leistungsstarken Editor mit vielen Funktionen, der normalerweise ein "ganzes Projekt" bearbeitet. Wie der Name schon sagt, handelt es sich nicht nur um einen Editor, sondern um eine umfassende "Entwicklungsumgebung".
 
-An IDE loads the project (which can be many files), allows navigation between files, provides autocompletion based on the whole project (not just the open file), and integrates with a version management system (like [git](https://git-scm.com/)), a testing environment, and other "project-level" stuff.
+Eine IDE lädt das Projekt (das aus vielen Dateien bestehen kann), ermöglicht das Navigieren zwischen Dateien, stellt die automatische Vervollständigung basierend auf dem gesamten Projekt (nicht nur der geöffneten Datei) bereit und lässt sich in ein Versionsverwaltungssystem (wie [git](https://git-scm.com/)), eine Testumgebung und andere Dinge auf "Projektebene" integrieren.
 
-If you haven't selected an IDE yet, consider the following options:
+Wenn du noch keine IDE ausgewählt haben, prüfen Sie die folgenden Optionen:
```
Comment (JoeGab) [inline 1-js/01-getting-started/3-code-editors/article.md]:
```suggestion
Wenn du noch keine IDE ausgewählt hast, zieh die folgenden Optionen in Betracht:
```

File: 1-js/01-getting-started/3-code-editors/article.md [hunk 14]
```diff
@@ -1,46 +1,46 @@
...
+- [WebStorm](http://www.jetbrains.com/webstorm/) (plattformübergreifend, kostenpflichtig).
 
-For Windows, there's also "Visual Studio", not to be confused with "Visual Studio Code". "Visual Studio" is a paid and mighty Windows-only editor, well-suited for the .NET platform. It's also good at JavaScript. There's also a free version [Visual Studio Community](https://www.visualstudio.com/vs/community/).
+Für Windows gibt es auch "Visual Studio", nicht zu verwechseln mit "Visual Studio Code". "Visual Studio" ist ein kostenpflichtiger und mächtiger Windows-Editor, der für die .NET-Plattform zugeschnitten ist. Er ist auch für JavaScript geeignet. Es gibt auch eine kostenlose Version: [Visual Studio Community](https://www.visualstudio.com/vs/community/).
 
-Many IDEs are paid, but have a trial period. Their cost is usually negligible compared to a qualified developer's salary, so just choose the best one for you.
+Viele IDEs sind kostenpflichtig, haben aber eine Testphase. Ihre Kosten sind im Vergleich zum Gehalt eines qualifizierten Entwicklers normalerweise vernachlässigbar, wählen Sie also einfach den Besten für sich aus.
```
Comment (JoeGab) [inline 1-js/01-getting-started/3-code-editors/article.md]:
```suggestion
Viele IDEs sind kostenpflichtig, haben aber eine Testphase. Ihre Kosten sind im Vergleich zum Gehalt eines qualifizierten Entwicklers normalerweise vernachlässigbar, wähle also einfach den Besten für dich aus.
```

File: 1-js/01-getting-started/3-code-editors/article.md [hunk 15]
```diff
@@ -1,46 +1,46 @@
...
+Sie werden hauptsächlich zum sofortigen Öffnen und Bearbeiten einer Datei verwendet.
 
-The main difference between a "lightweight editor" and an "IDE" is that an IDE works on a project-level, so it loads much more data on start, analyzes the project structure if needed and so on. A lightweight editor is much faster if we need only one file.
+Der Hauptunterschied zwischen einem "Lightweight-Editor" und einer "IDE" besteht darin, dass eine IDE auf Projektebene arbeitet, also beim Start viel mehr Daten lädt, die Projektstruktur bei Bedarf analysiert und so weiter. Ein Lightweight-Editor ist viel schneller, wenn wir nur eine Datei benötigen.
 
-In practice, lightweight editors may have a lot of plugins including directory-level syntax analyzers and autocompleters, so there's no strict border between a lightweight editor and an IDE.
+In der Praxis verfügen Lightweight-Editoren möglicherweise über eine Vielzahl von Plug-ins, darunter Syntaxanalyse und Autovervollständigung auf Verzeichnisebene. Daher gibt es keine exakte Grenze zwischen einem Lightweight-Editor und einer IDE.
 
-The following options deserve your attention:
+Die folgenden Optionen verdienen Ihre Aufmerksamkeit:
```
Comment (JoeGab) [inline 1-js/01-getting-started/3-code-editors/article.md]:
```suggestion
Die folgenden Optionen verdienen deine Aufmerksamkeit:
```

**Comment by MichFrie** [comment]
Hi 1HO, sorry for the confusion, please feel free to close my commit and use the other translation if that's helpful

**Comment by 1H0** [comment]
Hi @MichFrie!

You did everything right! You reserved the topic first, so it's fine! Just make sure to implement the changes requesten on your PR. Then we can merge your one!

I think i have to close this one now... Since it was translated twice...

**Comment by 1H0** [comment]
Duplicated translation... @MichFrie reserved the Topic properly, because of that we'll merge his one when the requested changes are implemented.

Sorry @higoka ... 😞 Make sure to properly claim the topics in the future!

**Pull request closed without merging**